src/widget/gui.cpp

```diff
@@ -322,25 +322,34 @@ QString GUI::passwordDialog(const QString& cancel, const QString& body)
 
 void GUI::_clearContacts()
 {
-    Nexus::getDesktopGUI()->clearContactsList();
+    Widget* w = Nexus::getDesktopGUI();
+    if (w)
+        w->clearContactsList();
```

Comment:
We don't update code just to update style. We write new code with new code style. And when we change some code, we update code style too.
> Why have it?

Because code written in one style easy to read. And sometime can help to avoid mistakes (like rule: brackets everywhere)